go.mod

```diff
@@ -2,10 +2,26 @@ module github.com/filecoin-project/go-f3
 
 go 1.20
 
-require github.com/stretchr/testify v1.8.2
+require (
+	github.com/filecoin-project/go-bitfield v0.2.4
```

Comment:
Please delete this requirement before merging, and ensure the indirect requirements and go.sum are reverted too.

Reply:
Done. Merging.